Makefile

```diff
@@ -1,4 +1,5 @@
 REGISTRY ?= quay.io/kubevirt
+OVS_CNI_GIT_VERSION ?= latest
```

Comment:
It is not connected to Git or version, in Docker it's called just "tag" 

IMAGE_TAG